packages/transmute-cli/src/commands/provision.js

```diff
@@ -1,46 +1,52 @@
 const run = require('./runner');
-const TRANSMUTE_KUBE_VERSION = process.env.TRANSMUTE_KUBE_VERSION ||  'v1.9.0';
-const MINIKUBE_CPU = process.env.MINIKUBE_CPU ||  '4';
-const MINIKUBE_MEMORY = process.env.MINIKUBE_MEMORY ||  '4096';
-const MINIKUBE_DISK = process.env.MINIKUBE_DISK ||  '100g';
-const MINIKUBE_PROFILE = process.env.MINIKUBE_PROFILE ||  'transmute-k8s';
+const TRANSMUTE_KUBE_VERSION = process.env.TRANSMUTE_KUBE_VERSION || 'v1.9.0';
+const MINIKUBE_CPU = process.env.MINIKUBE_CPU || '4';
+const MINIKUBE_MEMORY = process.env.MINIKUBE_MEMORY || '4096';
+const MINIKUBE_DISK = process.env.MINIKUBE_DISK || '100g';
+const MINIKUBE_PROFILE = process.env.MINIKUBE_PROFILE || 'transmute-k8s';
 
-export function minikube( dryrun, clusterName, minikubeDriver ) {
-  const minikube_start = 'minikube start '
-    + ' --kubernetes-version ' +  TRANSMUTE_KUBE_VERSION
-    + ' --disk-size ' +  MINIKUBE_DISK
-    + ' --cpus ' +  MINIKUBE_CPU
-    + ' --memory ' +  MINIKUBE_MEMORY;
-    + ' --profile ' +  MINIKUBE_PROFILE;
+module.exports.minikube = (dryrun, clusterName, minikubeDriver) => {
+  const minikube_start =
```

Comment:
I'm much more a fan of [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), but this seems long enough to want to have everything listed out like this.

Reply:
me too, save this for a refactor.